apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff

```diff
@@ -285,6 +300,11 @@
         <target>Создать профиль</target>
         <note>No comment provided by engineer.</note>
       </trans-unit>
+      <trans-unit id="Currently maximum supported file size is %@." xml:space="preserve">
+        <source>Currently maximum supported file size is %@.</source>
+        <target>Максимальный размер файла - %@.</target>
```

Comment:
the translation goes as following in android, maybe we should change it there as well
```suggestion
        <target>В настоящее время максимальный поддерживаемый размер файла составляет %@.</target>
```

Reply:
I don't think it's needed...